lib/openstudio-standards/standards/Standards.Space.rb

```diff
@@ -1147,9 +1135,9 @@ def space_add_daylighting_controls(space, remove_existing_controls, draw_dayligh
       sensor_2.setName("#{space.name} Daylt Sensor 2")
       sensor_2.setSpace(space)
       sensor_2.setIlluminanceSetpoint(daylight_stpt_lux)
-      sensor_2.setLightingControlType('Stepped')
-      sensor_2.setNumberofSteppedControlSteps(3) # all sensors 3-step per design
-      sensor_2.setMinimumInputPowerFractionforContinuousDimmingControl(0.3)
+      sensor_2.setLightingControlType(space_daylighting_control_type(space))
+      sensor_2.setNumberofSteppedControlSteps(3) unless space_daylighting_control_type(space) != 'Stepped' # all sensors 3-step per design
+      sensor_2.setMinimumInputPowerFractionforContinuousDimmingControl(space_daylighting_minimum_input_power_fraction(space))
```

Comment:
Lookup control method and fraction using methods instead of using hard-coded defaults.